breaking-bitcoin/2019/fud-perceived-vs-real-bitcoin-risks.md

```diff
@@ -98,7 +95,7 @@ EV: If people want new coins, changes to coins, it's very easy to do. It might b
+JS: The question assumes that bitcoin is primarily a technology. To me, it's money. For money, it's better if it doesn't change. It's about additive features that are backwards compatible, that's what a soft-fork is. Having a stable base layer is a feature not a bug. When people say hey ethereum is bitcoin 2.0 and eos is bitcoin 3.0, they have a completely skewed view of what this whole thing is. It's money, and you want it to stay stable. The idea that you need to evolve it or something like that is a completely misplaced notion that mostly comes from Silicon Valley where they see everything as a tech play when it is actually a monetary play instead. As far as soft-forks and whether they will activate, segwit was obviously a very difficult thing. Fortunately a lot of the haters have left, and one of the great things about bitcoin is that if you disagree you're free to leave and do your own thing. You're free to do that at any time. That's a feature, again, not a bug. A lot of people say, well now you have this toxic culture and everyone hates each other, but if you disagree enough then you can go and make your own thing. That's totally fine. In many ways, I expect the next soft-fork to be less difficult than the previous one. But who knows. It's possible that there's a much harder time doing soft-forks going forward. It's okay if it doesn't happen. The base layer works great. Would we like to have additional features so that lightning is much easier to implement and things like that? Yeah, absolutely. I would love taproot, Schnorr signatures, SIGHASH_NOINPUT and all these other things coming in. Is it necessary? Not exactly. It works great as it is. It's adding additional features to make it even better, but it's not necessary.
```

Comment:
```suggestion
JS: The question assumes that bitcoin is primarily a technology. To me, it's money. For money, it's better if it doesn't change. It's about additive features that are backwards compatible, that's what a soft-fork is. Having a stable base layer is a feature not a bug. When people say hey ethereum is bitcoin 2.0 and eos is bitcoin 3.0, they have a completely skewed view of what this whole thing is. It's money, and you want it to stay stable. The idea that you need to evolve it or something like that is a completely misplaced notion that mostly comes from Silicon Valley where they see everything as a tech play when it is actually a monetary play instead. As far as soft-forks and whether they will activate, segwit was obviously a very difficult thing. Fortunately a lot of the haters have left, and one of the great things about bitcoin is that if you disagree you're free to leave and do your own thing. You're free to do that at any time. That's a feature, again, not a bug. A lot of people say, well now you have this toxic culture and everyone hates each other, but if you disagree enough then you can go and make your own thing. That's totally fine. In many ways, I expect the next soft-fork to be less difficult than the previous one. But who knows. It's possible that there's a much harder time doing soft-forks going forward. It's okay if it doesn't happen. The base layer works great. Would we like to have additional features so that lightning is much easier to implement and things like that? Yeah, absolutely. I would love taproot, Schnorr signatures, SIGHASH\_NOINPUT and all these other things coming in. Is it necessary? Not exactly. It works great as it is. It's adding additional features to make it even better, but it's not necessary.
```